.specify/scripts/bash/check-prerequisites.sh

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
```

Comment:
if we go with this branch we should convert these to python. And let's add python scripts to the constitution.

Reply:
I really like the open-ness of the tools used too. It's hacky and feels unpolished but it's so easy to write the tools and commands that it's nice to have that ability to customize. Over time there may be particular tweaks that one realizes are appropriate for certain project types (size, complexity, idiosyncratic constraints etc.)